src/components/MovieCard/MovieCard.jsx

```diff
@@ -1,5 +1,44 @@
 import './MovieCard.scss';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ movie }) => (
+  <div className="MovieCard" data-cy="Movie">
```

Comment:
you can make destructuring from a movie
```suggestion
export const MovieCard = ({ movie }) => {
  const { ... } = movie;
  
  return (
    <div className="MovieCard" data-cy="Movie">
```